### PR TITLE
Configure mysql db

### PR DESCRIPTION
**Pull request by ismail-syed**
* Add `mysql2` gem
* Configures rails app to connect with mysql 
* Document setup

Fixes https://github.com/GraphQL-Query-Planner/webscale/issues/4

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,9 +7,6 @@
 # Ignore bundler config.
 /.bundle
 
-# Ignore the default SQLite database.
```
Comment (ismail-syed) [inline .gitignore:10]: Don't know what the `mysql` equivalents are that need to be ignored. We can `.gitignore` them as we see them.

Comment (justinkrol) [reply .gitignore:10]: SQLite creates a separate db per project, in the project directory, but MySQL is on its own server, so I don't think any MySQL database files will show up in the repo.

File: README.md [hunk 2]
```diff
@@ -36,3 +36,22 @@ $ rails server
...
+```bash
+$ brew install mysql
+```
+
+By default the mysql user is root with no password.
+
+Once installed, verify your environments connection to the database by creating the databases:
+
+```
+$ rake db:create
```
Comment (DerekStride) [inline README.md]: I had to do `mysql.server start` before this because mysql wasn't running for me.

**Review by DerekStride — APPROVED** [review]
LGTM one comment about starting mysql when it's not running.

File: README.md (hunk 2 above)
Comment (ismail-syed) [reply README.md]: Oh yah, good catch. 
I think the [recommended](https://gorails.com/setup/osx/10.12-sierra) command is `brew services start mysql`.
I'm gonna add `brew services start mysql` to the docs. `mysql.server start` wasn't working for me.

**Pull request merged**